cluster-proxy/src/main/java/com/sequenceiq/cloudbreak/clusterproxy/ConfigRegistrationRequest.java

```diff
@@ -60,29 +63,31 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-
         ConfigRegistrationRequest that = (ConfigRegistrationRequest) o;
-
-        return Objects.equals(clusterCrn, that.clusterCrn) &&
+        return useTunnel == that.useTunnel &&
+                Objects.equals(clusterCrn, that.clusterCrn) &&
                 Objects.equals(uriOfKnox, that.uriOfKnox) &&
                 Objects.equals(aliases, that.aliases) &&
                 Objects.equals(services, that.services) &&
-                Objects.equals(certificates, that.certificates);
+                Objects.equals(certificates, that.certificates) &&
+                Objects.equals(tunnels, that.tunnels);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(clusterCrn, uriOfKnox, aliases, services, certificates);
+        return Objects.hash(clusterCrn, uriOfKnox, aliases, services, certificates, useTunnel, tunnels);
```

Comment:
Should we include accountId here? It should not matter logically since clusterCrn already includes accountId. But feels better to use it than not.

Reply:
Done